core/templates/dev/head/pages/exploration_editor/simple_editor_tab/SimpleEditorShimServiceSpec.js

```diff
@@ -85,10 +81,10 @@ describe('Simple Editor Shim Service', function() {
     module('oppia');
     module(function($provide) {
       $provide.value('explorationTitleService', stubs.explorationTitleService);
-      $provide.value('explorationInitStateNameService', stubs.
-        explorationInitStateNameService);
-      $provide.value('explorationStatesService', stubs.explorationStatesService
-        );
+      $provide.value('explorationInitStateNameService',
```

Comment:
and I will reply to things from now on. and thanks for pointing me to wiki, some parts are still left for me to read.